framework/components/ecl-profile-topbars/ecl-profile-topbars.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * ECL Profile Topbar
+ * @define profile-topbar
+ */
+
+.ecl-profile-topbar {
+  margin: 0;
+}
+
+// This is esentially .section__group which can be used when refactored.
+.ecl-profile-topbar__wrapper {
```

Comment:
You're right, it's a perfect candidate for a re-usable class. It's `section__group` as of current state of the repo. So, i added them as a wrapper and added specifically the `section__group` name so that it's found in the code base and replaced when the latter is upgraded.